### DISCUSSION TITLE
Is it possible to dinamic change de icon for the main sidebar collapse?

### DISCUSSION THREAD

**Discussion by GilsonSBritto** [ColorlibHQ/AdminLTE#5205] [Q&A]
I want to make the icon for the sidebar collapse to be like an arrow that points in a specific direction when the sidebar is collapsed an when its not, like de card collapse icon on the example, can somebody help me with this?

![tela de exemplo](https://github.com/ColorlibHQ/AdminLTE/assets/86978710/a9b84a07-aa7f-454c-9b40-9d0d70bf4067)

**Comment by danny007in** [comment]
I'll check on that

**Reply by GilsonSBritto** [reply]
Did you find anything about it?

**Reply by danny007in** [reply]
can use different icon like bootstrap icon, fontawesome etc.. dynamic change using if else condition
